### DISCUSSION TITLE
Dynamic Expresso needs a collaborator

### DISCUSSION THREAD

**Discussion by davideicardi** [davideicardi/DynamicExpresso#177] [Q&A] [answered]
I'm looking for someone that can help me to keep Dynamic Expresso active.

I really appreciate the work done in the last year by @metoule !

@metoule, can you consider to join the Dynamic Expresso team? (for now composed just by me ;-) )
There is anyone else that want to help me?

Feel free to answer here or send me an email at `davide.icardi@gmail.com`.

(I have used the "Discussions" feature because I don't know how to contact you directly ...)

**Comment by metoule** [comment, marked as answer]
Hi Davide, 
as you can see, I'm pretty enthusiastic about Dynamic Expresso, so I'd be very happy to help you, and join the team :)
I'll also send you an email in case you prefer to discuss via mail.

**Reply by davideicardi** [reply]
Great! I will contact you via email to organize the details.
